frontend/packages/helm-plugin/src/actions/creators.ts

```diff
@@ -25,8 +25,8 @@ export const getHelmDeleteAction = (
       onSubmit: () => {
         return coFetchJSON.delete(
           `/api/helm/release/async?name=${releaseName}&ns=${namespace}&version=${releaseVersion}`,
-          null,
-          null,
+          undefined,
+          undefined,
```

Comment:
I realize this is probably needed due to the type declaration, but it seems like splitting hairs between `undefined` and `null`. Would it be better to allow `coFetchJSON` to accept `null`?